internal/compiler/generator/rust.rs

```diff
@@ -2539,6 +2540,17 @@ fn compile_builtin_function_call(
                 sp::Color::from_argb_u8(a, r, g, b)
             })
         }
+        BuiltinFunction::Hsv => {
+            let (h, s, v, a) =
+                (a.next().unwrap(), a.next().unwrap(), a.next().unwrap(), a.next().unwrap());
+            quote!({
+                let h: f32 = (#h as f32).max(0.).min(360.) as f32;
```

Comment:
Just curious... is there a specific reason to prefer `.max(0.).min(360.)` over `.clamp(0., 360.)`?

Reply:
I don't think there is a reason.

```suggestion
                let h: f32 = (#h as f32).clamp(0., 360.) as f32;
```